service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -28,7 +35,9 @@
 @Slf4j
 public class MigrateCaseController extends CallbackController {
     private static final String MIGRATION_ID_KEY = "migrationId";
-    private final CaseSubmissionService caseSubmissionService;
+    private static final String ID_FIRST_BUNDLE = "fbf05208-f5dd-4942-b735-9aa226d73a2e";
```

Comment:
would it be better to store them as uuids instead, you are doing all the comparisons later by converting the uuids to strings or the other way round